### ISSUE TITLE
selectInput inaccessible to screen readers

### ISSUE THREAD

**Issue by gtritchie** [rstudio/shiny#2847]
### System details

Output of `sessionInfo()`:

```
R version 3.6.3 (2020-02-29)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.4

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shiny_1.4.0.9002

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4.6         crayon_1.3.4         withr_2.1.2          packrat_0.5.0        digest_0.6.25       
 [6] later_1.0.0          mime_0.9             R6_2.4.1             jsonlite_1.6.1       xtable_1.8-4        
[11] magrittr_1.5         rlang_0.4.5          rstudioapi_0.11      promises_1.1.0       tools_3.6.3         
[16] tinytex_0.22         rsconnect_0.8.16     httpuv_1.5.2         xfun_0.13            fastmap_1.0.1     
```

### Example application
```R
library(shiny)

ui <- fluidPage(
   selectInput('xcol', 'X Variable', names(iris)),
   selectInput('ycol', 'Y Variable', names(iris),
               selected = names(iris)[[2]]),
   numericInput('clusters', 'Cluster count', 3, min = 1, max = 9)
)
server <- function(input, output) {}
shinyApp(ui = ui, server = server)
```

<img width="345" alt="screenshot of shiny app with two selectinput controls and a numericInput control" src="https://user-images.githubusercontent.com/10569626/80261849-76f4b900-8640-11ea-94cd-e5cc22692789.png">

### Describe the problem in detail
Run this app in browser, try to navigate/manipulate via screen reader. For my test case, I'm using Safari / VoiceOver on macOS, but the same issues manifest on Windows with NVDA and JAWS.

#### Problem One
- Tab to put focus on the first control ("X Variable")
- screen reader says: "X Variable, Edit Text"; this is wrong and suggests to user that they should type in some text here (which they cannot)
- here's what displayed at this point:

<img width="359" alt="screenshot of shiny app showing first select control dropped down" src="https://user-images.githubusercontent.com/10569626/80262065-20d44580-8641-11ea-8882-7e52dc65d00a.png">

#### Problem Two
- Nothing is said about the currently selected item

#### Problem Three
- Use up/down arrows to change selection (not that a screen reader-user would know they could do this)
- Nothing is announced as the selection changes

#### Problem Four
- With the popup shown on the second select-input instance, hit <enter> to accept the current selection
- Focus should return to that control's edit field, but instead if returns to the first control

### Expected

This control needs to implement the appropriate aria pattern, e.g. https://www.w3.org/TR/wai-aria-practices/#combobox

**Comment by jooyoungseo** [comment]
This issue comes from `selectize.js` accessibility. If you add `, selectize=FALSE` argument to the `selectInput()` function, your input will be made accessible because it will then use the native html5 `<select>` tag.

I will fix the `selectize.js` issue fundamentally soon.
